<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWidgets</class>
 <widget class="QMainWindow" name="MainWidgets">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>791</width>
    <height>518</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWidgets</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>3</number>
      </property>
      <widget class="QWidget" name="ptab1">
       <attribute name="title">
        <string>Tab 1</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="ptab2">
       <attribute name="title">
        <string>Tab 2</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="ptab3">
       <attribute name="title">
        <string>Tab 3</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="ptab4">
       <attribute name="title">
        <string>Tab 4</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>791</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
